packages/office-ui-fabric-react/src/components/TextField/docs/new/TextFieldBestPractices.md

```diff
@@ -0,0 +1,12 @@
+### Layout
+
+- Use a multiline text field when long entries are expected
+- Don’t place a TextField iin the middle of a sentence. For example, “Remind me in [textfield] weeks” should instead read, “Remind me in this many weeks [textfield]”.
+- Whenever possible, format the TextField for the expected entry. For example, four fields for a four-digit PIN, three fields for a phone number.
+
+### Content
+
+- Include a label at the top of the text field to communicate what information should be entered.
```

Comment:
Not sure the location needs to be mentioned, since the label automatically goes on top.